eclass/pam.eclass

```diff
@@ -29,15 +29,15 @@ inherit flag-o-matic
 dopamd() {
 	[[ -z $1 ]] && die "dopamd requires at least one argument"
 
-	if has pam ${IUSE} && ! use pam; then
+	if in_iuse pam && ! use pam; then
```

Comment:
This test occurs multiple times. Maybe define it as a function like this?
```
_pam_flag_disabled() {
	in_iuse pam && ! use pam
}
```
Also, must these be `if` blocks?
```
	_pam_flag_disabled && return 0
```